src/CyBy/Draw/MoleculeCanvas.idr

```diff
@@ -232,8 +232,8 @@ nextMol s =
       Rotating p       => rotateSelected (s.modifier == Shift) p s.posMol s.mol
       SetTempl t       => addTemplate s.posMol t s.mol
       RotTempl p t     => addTemplate p (rotateTempl False p s.posMol t) s.mol
-      RotTemplOvp ps t => addTemplateRot s.posMol (Right ps) t.graph s.imol
-      RotTemplAtm p t  => addTemplateRot s.posMol (Left p) t.graph s.imol
+      RotTemplOvp g t ps => addTemplateRot s.posMol (Right ?foops) t.graph s.imol
+      RotTemplAtm m t p  => addTemplateRot s.posMol (Left ?foop) t.graph s.imol
       Drawing Nothing  => addBond (s.modifier == Shift) s.posMol s.bond s.imol
```

Comment:
You are trying to tell Idris that the order of `g` and `m` is the same as the one of `s.imol`. Why should it believe you?